### PR TITLE
Feature/77012

### PR DESCRIPTION
**Pull request by diegolucasb**
#### What this PR does?
Creates a new component to hide its value

#### How should it be manually tested?
hideablecomponent

### CODE REVIEW

File: app/src/main/java/br/com/stone/emeraldcomponentsandroid/MainActivity.kt [hunk 1]
```diff
@@ -31,7 +32,9 @@ class MainActivity : AppCompatActivity() {
                 SpinnerActivity::class.java to "Spinner",
                 DateActivity::class.java to "Date Selector and Filter",
                 PagerActivity::class.java to "Pager",
-                LabelActivity::class.java to "Label")
+                LabelActivity::class.java to "Label",
+                HideableLabelActivity::class.java to "Hidealbe Label"
```
Comment (VictorGTdaCruz) [inline app/src/main/java/br/com/stone/emeraldcomponentsandroid/MainActivity.kt]:
```suggestion
                HideableLabelActivity::class.java to "Hideable Label"
```

File: emeraldcomponents/src/main/res/layout/widget_emerald_hideable_label.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        android:textSize="20sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="50.000,00" />
+
+    <View
+        android:id="@+id/hiddenView"
+        android:layout_width="0dp"
+        android:layout_height="24dp"
```
Comment (VictorGTdaCruz) [inline emeraldcomponents/src/main/res/layout/widget_emerald_hideable_label.xml]: Is it ok to have this height set like this? Have you tried with different view sizes?

File: emeraldcomponents/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <resources>
 
     <!--Emerald Colors-->
-    <color name="emerald_primary">#0DB14B</color>
+    <color name="emerald_primary">#088037</color>
```
Comment (VictorGTdaCruz) [inline emeraldcomponents/src/main/res/values/colors.xml]: Why is this here?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## Unreleased
+
+### Added
+- [EmeraldHideableLable component to hide value if is intended][issue-125]
```
Comment (VictorGTdaCruz) [inline CHANGELOG.md]:
```suggestion
- [EmeraldHideableLable component to hide value if desired][issue-125]
```

File: emeraldcomponents/src/main/res/values/colors.xml (hunk 3 above)
Comment (diegolucasb) [reply emeraldcomponents/src/main/res/values/colors.xml]: Because emerald has changed: https://www.figma.com/file/VxaCidggvWqzyc6cs1V3Km/Guia-de-estilo?node-id=65%3A1506

Maybe it is better to keed the old one

File: emeraldcomponents/src/main/res/layout/widget_emerald_hideable_label.xml (hunk 2 above)
Comment (diegolucasb) [reply emeraldcomponents/src/main/res/layout/widget_emerald_hideable_label.xml]: Good point :thinking:

File: emeraldcomponents/src/main/res/values/colors.xml (hunk 3 above)
Comment (VictorGTdaCruz) [reply emeraldcomponents/src/main/res/values/colors.xml]: You mean keep? I think this kind of change should be done carefully, since other projects are using emerald colors in their layouts

Comment (VictorGTdaCruz) [reply emeraldcomponents/src/main/res/values/colors.xml]: For example, we already have a purple (emerald_secondary). If the new purple is added and used for now on, we have to review other projects that use the colors to update (Is this a breaking change?)

Comment (diegolucasb) [reply emeraldcomponents/src/main/res/values/colors.xml]: As we are not gonna use this color right now, it is better to leave the old one

Comment (diegolucasb) [reply emeraldcomponents/src/main/res/values/colors.xml]: As the change is not set yet, let's keep this way and wait for the official design documentation
